doc/source/structures/collections/lexicon.rst

```diff
@@ -176,7 +172,7 @@ Structure
     
 .. attribute:: Lexicon:LENGTH
 
```

Comment:
a couple of attributes did not get the type revision: `casesensitive` and `case`